projects/tags-input-demo/src/app/app.component.html

```diff
@@ -304,51 +304,56 @@
 
   <div class="content" role="main">
     <form [formGroup]="simpleForm" class="main__form">
-
       <div>
         <h3 class="main__form__tt">Nested Dropdown</h3>
-      <ti-angular-tags-input formControlName="users" class="main__form__ti"
+        <ti-angular-tags-input formControlName="users" class="main__form__ti"
         [tagsData]="sampleDataNested"
         (tagAdded)="onTagAdded($event)"
         (valueChanged)="onValChanged($event)"
         [config]="nestedTagsInputConfig"
         [dropDownTemplate]="ddNestedTemplate"
         >
-      </ti-angular-tags-input>
+        </ti-angular-tags-input>
       </div>
-
       <div>
         <h3 class="main__form__tt">Simple Dropdown</h3>
-      <ti-angular-tags-input formControlName="users" class="main__form__ti"
+        <ti-angular-tags-input formControlName="users" class="main__form__ti"
         [tagsData]="sampleDataSimple"
         (tagAdded)="onTagAdded($event)"
         (valueChanged)="onValChanged($event)"
         [config]="simpleTagsInputConfig"
         >
-      </ti-angular-tags-input>
+        </ti-angular-tags-input>
       </div>
-
       <div>
-      <h3 class="main__form__tt">Image Dropdown</h3>
-      <ti-angular-tags-input formControlName="users" class="main__form__ti"
-      [tagsData]="sampleDataImage"
-      (tagAdded)="onTagAdded($event)"
-      (valueChanged)="onValChanged($event)"
-      [config]="ImageTagsInputConfig"
-      [dropDownTemplate]="userTagItemTemplate"
-      >
-    </ti-angular-tags-input>
-    </div>
-      
-
+        <h3 class="main__form__tt">Image Dropdown</h3>
+        <ti-angular-tags-input formControlName="users" class="main__form__ti"
+        [tagsData]="sampleDataImage"
+        (tagAdded)="onTagAdded($event)"
+        (valueChanged)="onValChanged($event)"
+        [config]="imageTagsInputConfig"
+        [dropDownTemplate]="userTagItemTemplate"
+        >
+        </ti-angular-tags-input>
+      </div>
     </form>
   </div>
-
-  <ng-template #userTagItemTemplate let-item="item" let-tagClass="tagClass" let-config="config" let-closeClicked="closeClicked">
-    <tid-users-dropdown-template>
-    </tid-users-dropdown-template>
+  <ng-template #userTagItemTemplate let-config="config">
+      <div class="users-list" *ngFor="let item of sampleDataImage">
```

Comment:
We're running `*ngFor` on the div `.users-list` which means this div is being copied as many times as the length of `sampleDataImage`. Instead, we should have `*ngFor`  on the `users-list__item` div.
What you can do even better is create a component name `users-list-item`. 
That way, you can encapsulate the html and also the css styles within that component.
Then use something like

```html
<ng-template #userTagItemTemplate let-config="config">
  <div class="users-list">
    <tid-users-list-item
      *ngFor="let item of sampleDataImage"
      [item]="item"
      [config]="config">

    </tid-users-list-item>
  </div>
</ng-template>
```